fenix/app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -136,44 +138,6 @@ class SyncIntegrationTest {
         }
     }
 
-    // Bookmark item Fenix -> Desktop
-    @Test
```

Comment:
This test is disabled for now and I think that all tests in this test target are not running... We want to re-enable them again though. It is not a regular test since it will run part on android device and part on desktop so that we verify the E2E environment as we are doing with these tests on iOS.
That said, I would say it is fine to remove it from now